spec/controllers/content_items_controller_spec.rb

```diff
@@ -2,26 +2,77 @@
 
 RSpec.describe ContentItemsController, type: :controller do
   describe "GET #index" do
-    let(:organisation) { create(:organisation_with_content_items) }
+    let(:organisation) { create(:organisation_with_content_items, content_items_count: 4) }
 
-    before do
-      get :index, params: { organisation_id: organisation }
-    end
+    context "find by organisation" do
+      before do
+        get :index, params: { organisation_id: organisation }
+      end
 
-    it "returns http success" do
-      expect(response).to have_http_status(:success)
-    end
+      it "returns http success" do
+        expect(response).to have_http_status(:success)
+      end
 
-    it "assigns current organisation" do
-      expect(assigns(:organisation)).to eq(organisation)
-    end
+      it "assigns current organisation" do
+        expect(assigns(:organisation)).to eq(organisation)
+      end
+
+      it "assigns list of content items" do
+        expect(assigns(:content_items)).to eq(organisation.content_items)
+      end
 
-    it "assigns list of content items" do
-      expect(assigns(:content_items)).to eq(organisation.content_items)
+      it "renders the :index template" do
+        expect(subject).to render_template(:index)
+      end
     end
 
-    it "renders the :index template" do
-      expect(subject).to render_template(:index)
+    context "sorts with parameters" do
+      context "in ascending order" do
```

Comment:
Also, by setting the number of `content_items_count` to four we are slowing down the test as the before clause applies to every test.